src/inim.nim

```diff
@@ -239,14 +241,14 @@ proc hasIndentTrigger*(line: string): bool =
 
 proc runForever() =
     while true:
+
         # Read line
         try:
-            currentExpression = readLineFromStdin(getPromptSymbol()).strip
-        except IOError:
-            bufferRestoreValidCode()
-            indentLevel = 0
-            tempIndentCode = ""
-            continue
+          stdout.write(getPromptSymbol())
+          stdout.flushFile()
+          currentExpression = stdin.readLine()
```

Comment:
Can you think why `readLineFromStdin` was used before? My solution seems almost TOO simple...

Reply:
I think the main reason is to leverage the features from `linenoise`, like navigating with the arrow keys

Reply:
oh! Well spotted. Can we wrap this simply with `rlwrap`? That does the trick, but we'd need to vendor that in...